setup.cfg

```diff
@@ -1,2 +1,12 @@
 [bdist_wheel]
 universal = true
+
+[mypy]
+python_version = 3.7
+namespace_packages=True
```

Comment:
We're not using namespace packages so I don't think we need this. Can we also not cherry-pick individual configs but just make it globally strict?

Reply:
can remove this as well. I'm not aware of a way to make it globally strict in [mypy](https://mypy.readthedocs.io/en/stable/config_file.html)

Reply:
Just pass in the --strict flag in tox.ini invocation?